.github/workflows/performance.yaml

```diff
@@ -7,6 +7,10 @@ env:
   INFLUX_TOKEN: ${{ secrets.INFLUX_TOKEN }}
   WT_METRICS_DIR: "${{ github.workspace }}/telegraf/metrics"
 
+concurrency:
```

Comment:
I think we want to prevent this running more than once at the same time?

Reply:
It's more about not being able to trust the results if mutliple runs are attempted concurrently on the machine